orderbook/openapi.yml

```diff
@@ -383,8 +383,10 @@ components:
           type: string
           enum:
             [
+                Cancelled,
```

Comment:
This is not an error is it? It's the success case on the Result enum and as such shouldn't be listed here.